proposal/shared-resources.d.ts.md

```diff
@@ -147,10 +147,11 @@ export function initAsyncCompiler(): Promise<AsyncCompiler>;
 
 #### Compiler
 
-An instance of the synchronous Compiler.
+An instance of the synchronous Compiler. If the constructor is directly invoked
```

Comment:
"invoked" -> "constructed"

Shouldn't these also have private constructors?

Reply:
Updated in 3a4ca475e37db9ac3efa231df923766fe156ccad